commcare_connect/opportunity/models.py

```diff
@@ -426,7 +426,7 @@ class Meta:
 class Payment(models.Model):
     amount = models.PositiveIntegerField()
     amount_usd = models.DecimalField(max_digits=10, decimal_places=2, null=True)
-    date_paid = models.DateTimeField(auto_now_add=True)
+    date_paid = models.DateTimeField(default=datetime.datetime.utcnow)
```

Comment:
Without this it's not possible to override `date_paid` while doing `Payment.objects.create()`